### PR TITLE
Lesson5 HW

### PR DESCRIPTION
**Pull request by serghei-rubailo**

### CODE REVIEW

File: lesson5.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#make all the selected strings downcase
+names.map(&:downcase!)
+
+#using the last array, create a hash with key: name and value: name size
+names_hash = {}
+names.map {|index| names_hash[index] = index.size}
+
+#using the hash, get the longest name (this can be done without if using hash and array
+#methods, use the links from the previous slide)
+longest_name = names_hash.keys.sort{|a,b| a.size <=> b.size}.reverse!.first
```
Comment (jeneamocan) [inline lesson5.rb]: You can try the `max_by` method

Comment (serghei-rubailo) [reply lesson5.rb]: Thank you!
Yes, max_by looks more readable and works faster.

**Comment by jeneamocan** [comment]
Nice job!

**Comment by jeneamocan** [comment]
Oh, just one thing: can you place your homework script into a folder with your name? This will make it easier to track your progress. Add new files related to your homework to that folder in all other pull requests.

**Comment by serghei-rubailo** [comment]
> Oh, just one thing: can you place your homework script into a folder with your name? This will make it easier to track your progress. Add new files related to your homework to that folder in all other pull requests.

Ok. Done.
lesson5.rb HW file was moved to "rubailo_s" directory.

**Pull request merged**